go.mod

```diff
@@ -33,11 +33,11 @@ require (
 	github.com/jaypipes/ghw v0.7.0
 	github.com/karalabe/usb v0.0.0-20190919080040-51dc0efba356 // indirect
 	github.com/livepeer/livepeer-data v0.2.0
-	github.com/livepeer/lpms v0.0.0-20210806125031-9fdbf80c8575
+	github.com/livepeer/lpms v0.0.0-20210910093742-d3f7d75f2b5d
```

Comment:
Is that intentional that it points to branch (not to master)?

Reply:
Will update this PR once https://github.com/livepeer/lpms/pull/260 is merged